client/index.js

```diff
@@ -8,13 +8,13 @@ const handleMousemoveEvent = require('./events/mousemove')
 
 const baseUrl = process.env.COLLECTOR_BASE || 'http://localhost:3000/'
 
-function ic (...rest) {
+function customerCode (...rest) {
```

Comment:
A function name should be verb it may be something like `init` according to its duty